src/Domain/Regulation/Repository/RegulationOrderRecordRepositoryInterface.php

```diff
@@ -13,8 +13,8 @@ public function add(RegulationOrderRecord $regulationOrderRecord): RegulationOrd
 
     public function findOneByUuid(string $uuid): ?RegulationOrderRecord;
 
-    public function findRegulationsByOrganizations(
-        array $organizations,
+    public function findAllRegulations(
+        ?array $organizations,
         int $maxItemsPerPage,
         int $page,
         bool $isPermanent,
```

Comment:
On peut en profiter pour corriger le nom du paramètre, `$organizationUuids`

```suggestion
        int $maxItemsPerPage,
        int $page,
        bool $isPermanent,
        ?array $organizationUuids = null,
```